cmd/login.go

```diff
@@ -45,6 +48,42 @@ var loginCmd = &cobra.Command{
 		}
 		// end version check
 
+		if email == "" && password == "" && sso != "" {
+			sessionToken, err := http.CreateDeviceAuthorization()
+			if err != nil {
+				log.Fatalf("error: %v", err)
+			}
+
+			sessionJWT, _ := jwt.Parse(sessionToken, nil)
+
+			var deviceIdentifier string
+			if sessionJWT != nil {
+				claims := sessionJWT.Claims.(jwt.MapClaims)
+				deviceIdentifier = fmt.Sprint(claims["identifier"])
+			}
+
+			fmt.Println(`please login with the link below:`)
+			fmt.Printf("%s/login?device_identifier=%v\n", http.WebUrl(), deviceIdentifier)
+
+			for {
+				token, err := http.GetDeviceAuthorization(sessionToken)
+				if err != nil {
+					if errors.Is(err, http.ErrUnauthorized) {
+						log.Fatalf("We coudn't log you in, your session is expired or you not authorized to perform this action: %v", err)
+					}
+
+					log.Fatalf("We coudn't log you in, make sure that you properly logged in the link above: %v", err)
```

Comment:
change you change this to:
We couldn't log you in, make sure that you  are properly logged in using the link above